src/jit/codegenarm64.cpp

```diff
@@ -3119,8 +3125,6 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
 
     case GT_PUTARG_STK:
         {
-            noway_assert(targetType != TYP_STRUCT);
```

Comment:
Probably not - it would make sense to have a putarg_stk of a SIMD type, for example, because it could be stored directly onto the stack. This would generally hold for any "enregisterable" struct.